README.md

```diff
@@ -47,3 +47,50 @@ make install-packages opts="--with dev"
 # Install with specific extras
 make install-packages opts="--extras 'feature1 feature2'"
 ```
+
+### Usage
+
+#### Installation
+
+1. Add `"common.core"` to `INSTALLED_APPS` in your settings module.
+This enables the `start` management command.
+
+2. Add `"common.gunicorn.middleware.PrometheusGunicornLoggerMiddleware"` to `MIDDLEWARE` in your settings module. This enables the `path` label for Prometheus HTTP metrics.
+
+#### Metrics
+
+Flagsmith uses Prometheus to export backend metrics.
+
+The following default metrics are exposed:
+
+- `flagsmith_build_info`: Gauge with a constant value of `1.0`. Labeled with `ci_commit_sha` and `version`.
+- `http_request_duration_seconds`: Histogram labeled with `method`, `path`, and `response_status`.
+- `http_requests_total`: Counter labeled with `method`, `path`, and `response_status`.
```

Comment:
I would also suggest having a namespace for the metrics. Currently we only track metrics in the API, but we will track task processor metrics as well. Each of those should have their own namespace (and possibly a global `flagsmith` namespace as well).